src/rp2_common/pico_sha256/sha256.c

```diff
@@ -183,3 +194,10 @@ void pico_sha256_finish(pico_sha256_state_t *state, sha256_result_t *out) {
     }
     pico_sha256_unlock(state);
 }
+#endif
+
+#if defined(__ZEPHYR__)
+void pico_sha256_write_padding(pico_sha256_state_t *state) {
+	write_padding(state);
+}
+#endif
```

Comment:
The `#if !defined(__ZEPHYR__)` is useful to keep merging new SDK releases. However, I don't think the inverse is required.

```suggestion
void pico_sha256_write_padding(pico_sha256_state_t *state) {
	write_padding(state);
}
```

Reply:
It's true that no one else had done it this way, so I reflected it.